### ISSUE TITLE
Also install package updates

### ISSUE THREAD

**Issue by stdevel** [stdevel/katprep#33]
Beginning with Foreman 1.14.3, non-errata package updates can be triggered/installed using the UI - would be handy if ``katprep_maintenance`` could do the same.

**Comment by stdevel** [comment]
Unfortunately, the Foreman API currently offers no call for this.

**Comment by stdevel** [comment]
Things have changed, yas: https://theforeman.org/plugins/katello/3.5/api/apidoc/v2/host_packages.html

```
PUT /api/hosts/:host_id/packages/upgrade_all 
Update packages remotely
```

**Comment by stdevel** [comment]
Investigating in branch `feature-pkg_updates-issue33`

**Comment by stdevel** [comment]
Was thinking about adding an exclude parameter to `katprep_maintenance` - unfortunately I found no way to implement this. The API call `upgrade_all` upgrades all packages, while `upgrade` requires a list of package IDs. For this it would be necessary to have a list of "*upgradable*" packages for a particular host - and there is no such API call.

**Comment by stdevel** [comment]
In the web interface it is possible to list available package upgrades, so there must be a way to retrieve this information. Gonna ask the developers.

**Comment by stdevel** [comment]
In the web interface it is possible to list available package upgrades, so there must be a way to retrieve this information. Gonna ask the developers.

**Comment by stdevel** [comment]
Added code for installing all available package upgrades in commit 63ba47a6daaf19bd9e17f7b5f1e24ec7a795a5e5.
Successfully tested with Foreman 1.17.1 and Katello 3.6.
